fdbserver/ConfigDatabaseUnitTests.actor.cpp

```diff
@@ -733,6 +781,7 @@ TEST_CASE("/fdbserver/ConfigDB/Transaction/CompactThenGetConfigClasses") {
 	return Void();
 }
 
+// TODO: The below tests seem to always segfault
```

Comment:
That's strange, I don't see anything in your PR that should cause this. These tests can be commented out for now and reenabled in a future PR.

Reply:
They also seem to segfault on master (at least `/fdbserver/ConfigDB/Transaction/GetKnobs`, I didn't test the others). The issue is pretty strange too, it's when using `sscanf` to parse the string "2" into an integer. I debugged a little, but the issue didn't immediately jump out to me. I'll double check which of these tests pass and which segfault and remove the broken ones for now.

Reply:
Oh, this issue is fixed here: https://github.com/apple/foundationdb/pull/5329.

Reply:
Good to know! Will rebase